content/en/docs/v3.6/learning/api.md

```diff
@@ -48,16 +50,16 @@ message ResponseHeader {
 }
 ```
 
-* Cluster_ID - the ID of the cluster generating the response.
-* Member_ID - the ID of the member generating the response.
-* Revision - the revision of the key-value store when generating the response.
-* Raft_Term - the Raft term of the member when generating the response.
+* `cluster_id` - the ID of the cluster generating the response.
+* `member_id` - the ID of the member generating the response.
+* `revision` - the revision of the key-value store when generating the response.
+* `raft_term` - the Raft term of the member when generating the response.
```

Comment:
This is a pretty substantial reformatting of the file.  It's an improvement, but you should mention it in the PR description since it's most of the actual line-count for the patch.  

Also, it means we need to follow-up and see if there are other docs that reference the API and make sure they use the same formatting.